cli_tools/common/utils/daisy/daisy_utils_test.go

```diff
@@ -246,7 +246,7 @@ func createWorkflowWithCreateDiskImageAndIncludeWorkflow() *daisy.Workflow {
 				},
 				{
 					Disk: compute.Disk{
-						SourceImage: "projects/compute-image-tools/global/images/family/debian-10-worker",
+						SourceImage: "projects/compute-image-tools/global/images/debian-9-worker-v20200616",
```

Comment:
projects/compute-image-tools/global/images/family/debian-9-worker

Reply:
talked offline -- switched to `projects/compute-image-tools/global/images/family/debian-9-worker`